### PR TITLE
Iteration 2

### PR DESCRIPTION
**Pull request by BHaddad1**
Rough draft of function for iteration 2, some work for iteration 3 and lots of notes.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -133,7 +134,10 @@ takeMeBackButtonFromMake.addEventListener('click', goBackToMainMake);
 backToMainButtonFromSaved.addEventListener('click', goBackToMainSaved);
 makeOwnPosterButton.addEventListener('click', showMakeForm);
 //Iteration 2
-showMyPosterButton.addEventListener('click', showUserPoster);
+showMyPosterButton.addEventListener('click', function(){
```
Comment (baut-jc) [inline src/main.js:137]: cool. syntaxing two functions in this event clicker's argument :)

File: src/main.js [hunk 2]
```diff
@@ -189,27 +193,43 @@ function goBackToMainSaved(){
...
 }
 // Iteration 3
-function pushUserPosterToArray(){
+function saveUserPoster(){
+  var newUserImage = inputFieldForImage.value;
+  var newUserTitle = inputFieldForTitle.value;
+  var newUserQuote = inputFieldForQuote.value;
+  var newUserPoster = new Poster(newUserImage, newUserTitle, newUserQuote);
+
+  savedPosters.push({newUserPoster})
```
Comment (baut-jc) [inline src/main.js:216]: awesome!
instantiating a new poster class and pushing the entire object with it's properties, by instatiating the poster class and pushing it into the savedPosters array.

File: src/main.js (hunk 1 above)
Comment (BHaddad1) [reply src/main.js:137]: yeah! Pretty sure it was you who asked a question about that in class yesterday, right?

File: src/main.js [hunk 3]
```diff
@@ -189,27 +193,43 @@ function goBackToMainSaved(){
...
-  // images.push(newUserImage);
-  // titles.push(newUserTitle);
-  // quotes.push(newUserQuote);
-  // goBackToMainMake();
+  event.preventDefault()
+  var newUserImage = inputFieldForImage.value;
+  var newUserTitle = inputFieldForTitle.value;
+  var newUserQuote = inputFieldForQuote.value;
+  var newUserPoster = new Poster(newUserImage, newUserTitle, newUserQuote);
+  image.src = newUserImage;
```
Comment (baut-jc) [inline src/main.js:201]: replacing the values that will be invoked from the set event clicker.

File: src/main.js [hunk 4]
```diff
@@ -189,27 +193,43 @@ function goBackToMainSaved(){
...
+
+  savedPosters.push({newUserPoster})
   // create variables for user input fields query selectors
   // push variables into arrays
   // push new poster to saved posters arrays by calling the showUserPoster function?
   // create new object to push into the saved posters arrays
   //
 }
+
+function displayUserPoster(){
```
Comment (baut-jc) [inline src/main.js:224]: where will this be invoked?
an event listener? this function looks a lot like the showUserPoster.

...but this is a draft and we can definitely refactor later an when we get through the iterations.

File: src/main.js [hunk 5]
```diff
@@ -189,27 +193,43 @@ function goBackToMainSaved(){
...
+  var newUserImage = inputFieldForImage.value;
+  var newUserTitle = inputFieldForTitle.value;
+  var newUserQuote = inputFieldForQuote.value;
+  var newUserPoster = new Poster(newUserImage, newUserTitle, newUserQuote)
+  //display newUserPoster to grid
+  //newUserPoster is an object
+}
+
```
Comment (baut-jc) [inline src/main.js:232]: Overall.

This is an amazing progress!!
I'll see you around.

**Review by baut-jc — APPROVED** [review]
I've left some comments with my understanding of the codes you've implemented.

Overall,
This is an amazing progress!!
I'll see you around.

**Pull request merged**